### DISCUSSION TITLE
Resolving Merge Conflicts

### DISCUSSION THREAD

**Discussion by caheredia** [extrawurst/gitui#1563] [Ideas]
Is there a method to handle merge conflicts with gitUI? I know [Lazygit](https://blog.diegodev.com/git-managment-with-lazygit/) allows you to choose the specific hunks from the working branch and Master in the event of a conflict. 

My favorite method for handling conflicts is [fugitves/vimdiff](http://vimcasts.org/episodes/fugitive-vim-resolving-merge-conflicts-with-vimdiff/) method of displaying a 3-way diff for reconciling changes.

If gitUI has a merge conflict workflow maybe it should be added to the README.

**Comment by extrawurst** [comment]
We have no dedicated conflict resolve workflow
